trlx/data/configs.py

```diff
@@ -196,6 +196,9 @@ class TrainConfig:
 
     :param seed: Random seed
     :type seed: int
+
+    :param minibatch_size: Size of model input during one forward pass. Must divide batch size
```

Comment:
(very pedantic) nit, feel free to ignore: usually I've heard this called _micro_ batch, with minibatch referring to what we usually call "a batch" (to distinguish from a single batch of the whole dataset)